diff -Naur gsettings-desktop-schemas-3.10.1.orig/schemas/org.gnome.desktop.interface.gschema.xml.in.in gsettings-desktop-schemas-3.10.1/schemas/org.gnome.desktop.interface.gschema.xml.in.in
--- gsettings-desktop-schemas-3.10.1.orig/schemas/org.gnome.desktop.interface.gschema.xml.in.in	2013-08-12 21:41:25.000000000 +0700
+++ gsettings-desktop-schemas-3.10.1/schemas/org.gnome.desktop.interface.gschema.xml.in.in	2013-11-07 22:51:05.515262110 +0700
@@ -8,7 +8,7 @@
       </_description>
     </key>
     <key name="enable-animations" type="b">
-      <default>true</default>
+      <default>false</default>
       <_summary>Enable Animations</_summary>
       <_description>
         Whether animations should be displayed. Note: This is a global key,
@@ -83,21 +83,21 @@
       </_description>
     </key>
     <key name="icon-theme" type="s">
-      <default>'gnome'</default>
+      <default>'Komodo'</default>
       <_summary>Icon Theme</_summary>
       <_description>
         Icon theme to use for the panel, nautilus etc.
       </_description>
     </key>
     <key name="gtk-theme" type="s">
-      <default>'Adwaita'</default>
+      <default>'Manokwari'</default>
       <_summary>Gtk+ Theme</_summary>
       <_description>
         Basename of the default theme used by gtk+.
       </_description>
     </key>
     <key name="gtk-key-theme" type="s">
-      <default>'Default'</default>
+      <default>'Manokwari'</default>
       <_summary>Gtk+ Keybinding Theme</_summary>
       <_description>
         Basename of the default keybinding theme used by gtk+.
